sys/kern/main.c

```diff
@@ -57,24 +70,19 @@ static __noreturn void start_init(__unused void *arg) {
   error = session_enter(p);
```

Comment:
Is it mandatory to enter a session before we enter the user space for the first time?

Reply:
Actually it is not mandatory, provided that the session is created by the `init` process. For example in NetBSD it's on the side of `init` process to create a session - http://bxr.su/NetBSD/sbin/init/init.c#249.c. Therefore I changed the way the session is created in the case where we run `/sbin/init` - https://github.com/cahirwpz/mimiker/pull/1390/commits/567449066ac930d0572665fe6d6c1887c284b439